include/hermite_multidimensional.hpp

```diff
@@ -48,106 +48,108 @@ ullint vec2index(std::vector<int> &pos, int resolution) {
 
 }
 
+namespace libwalrus {
+
 /**
- * Returns the indices of the tensor corresponding to a given element
+ * Returns photon number statistics of a Gaussian state for a given covariance matrix `mat`.
+ * as described in *Multidimensional Hermite polynomials and photon distribution for polymode mixed light*
+ * [arxiv:9308033](https://arxiv.org/abs/hep-th/9308033).
+ *
+ * This implementation is based on the MATLAB code available at
+ * https://github.com/clementsw/gaussian-optics
  *
- * @param val
- * @param base
- * @param n
+ * @param mat a flattened vector of size \f$2n^2\f$, representing an
+ *       \f$2n\times 2n\f$ row-ordered symmetric matrix.
+ * @param d a flattened vector of size \f$2n\f$, representing the first order moments.
+ * @param resolution highest number of photons to be resolved.
  *
- * @return tensor index
  */
-std::vector<int> find_rep(int val, int base, int n) {
```

Comment:
How come this function was deleted?

Reply:
It is not needed anymore. It was only used by renomarlize which is gone.